README.md

```diff
@@ -19,6 +19,13 @@ cd iotjs
 tools/build.py
 ```
 
+### How to test
```

Comment:
For consistency, it would be better to change 'test' to 'Test' or 'Build' to 'build'(line 16).